client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -19,11 +15,8 @@ export default function Dashboard(): JSX.Element {
 
   const { loggedInUser } = useAuth();
 
-  const history = useHistory();
-
-  if (loggedInUser === undefined) return <CircularProgress />;
   if (!loggedInUser) {
-    history.push('/login');
+    //typescript is throwing errors without returning this.
```

Comment:
What errors? Submit a bug report in the slack channel

Reply:
Fixed